kge/job/train.py

```diff
@@ -109,8 +118,8 @@ def __init__(
         self.post_valid_hooks: List[Callable[[Job, Dict[str, Any]], Any]] = []
 
         #: Hooks run after training
-        #: Signature: job, trace_entry
-        self.post_train_hooks: List[Callable[[Job, Dict[str, Any]], Any]] = []
+        #: Signature: job
+        self.post_train_hooks: List[Callable[[Job], Any]] = []
```

Comment:
We not pass in the training trace? Also, that could be a post_run_hook on Job level that gets the job and the output of the run method

Reply:
but that one is from the last epoch, we can always call job.config.trace(), right?